styles.css

```diff
@@ -1,52 +1,96 @@
 h1 {
-    font-family: 'Luminari';
-    color: blue;
-    font-size: 30px;
+  font-family: "Luminari";
+  color: blue;
+  font-size: 30px;
 }
 
 ul {
-    list-style-type: none;
-}
-
-ol {
-    font-weight: bold;
+  list-style-type: none;
 }
```

Comment:
I think. you'll need to restore this deletion once you remove the solution code from this file